openmls/src/ciphersuite/crypto/rust_crypto_provider.rs

```diff
@@ -0,0 +1,214 @@
+//! # Evercrypt Crypto Provider
+//!
+//! Use evercrypt for all crypto operations.
+
+// XXX: ed25519-dalek depends on an old version of rand.
+//      https://github.com/dalek-cryptography/ed25519-dalek/issues/162
+// extern crate ed25519_dalek;
+extern crate rand_07;
```

Comment:
Isn't this pre 2018 edition syntax? Or is this needed for `ed25519_dalek` to work?

Reply:
yeah, dalek requires an old version and this is the "nicest" way to get it in

Reply:
Actually... I'm not sure why this is still in here. I probably forgot about it 🤔